server/app.py

```diff
@@ -204,10 +210,17 @@ async def submit_streaming(request: SubmitStreamingRequest):
     serialized_agent = pickle.dumps(request.agent)
 
     logger.info(f"Submitting task {task.name} with agent {serialized_agent}")
-    result = task.delay(serialized_agent=serialized_agent)
-    print(f"Task {task.name} submitted with job_id {result.id}")
+    input_result = task.delay(serialized_agent=serialized_agent)
+    input_job_id = input_result.id
+    logger.info(f"Task {task.name} submitted with job_id {input_job_id}")
 
-    return Response[JobCreated](data=JobCreated(job_id=result.id))
+    task = process_streaming_output
+    logger.info(f"Submitting task {task.name}")
+    output_result = task.delay(job_id=input_job_id, result_handler=request.result_handler)
+    output_job_id = output_result.id
+    logger.info(f"Task {task.name} submitted with job_id {output_job_id}")
+
+    return Response[JobCreated](data=JobCreated(job_id=input_job_id))
```

Comment:
I know we didn't quite resolve this, but I think we should be returning `output_job_id` here, right?

Reply:
As of now definitely not , we need this job ID on LSE side to submit batches to the correct Kafka queue